test/storage.test.js

```diff
@@ -310,7 +310,7 @@ describe('storage.js', () => {
                 .get('/photo/elephant.png')
                 .reply(200, 'ok');
 
-            const watermark = await getWatermark(params, inDirectory);
+            const watermark = await getSource(params, inDirectory, true, 'watermark');
```

Comment:
I find it confusing that 'watermark' here is the file's basename. Reading this without knowing the download code, I would think 'watermark' is an enum or constant value to indicate we are actually downloading a watermark. Even more because there's just a 'true'boolean as param just before.